mgmt_cli_test.py

```diff
@@ -216,34 +253,27 @@ def test_backup_feature(self):
         with self.subTest('Test Backup end of space'):  # Preferably at the end
             self.test_enospc_during_backup()
 
-    def update_config_file(self):
-        # FIXME: add to the nodes not in the same region as the bucket the bucket's region
-        # this is a temporary fix, after https://github.com/scylladb/mermaid/issues/1456 is fixed, this is not necessary
-        config_file = '/etc/scylla-manager-agent/scylla-manager-agent.yaml'
-        self.region = self.params.get('region_name').split()
-        self.bucket_name = self.params.get('backup_bucket_location').split()[0]
-        for node in self.db_cluster.nodes:
-            mgmt.update_config_file(node=node, region=self.region[0], config_file=config_file)
-        self.is_cred_file_configured = True
-
     def create_ks_and_tables(self, num_ks, num_table):
         # FIXME: beforehand we better change to have RF=1 to avoid restoring content while restoring replica of data
         table_name = []
-        for keyspace in range(num_ks):
-            self.create_keyspace(f'ks00{keyspace}', 1)
-            for table in range(num_table):
-                self.create_table(f'table00{table}', keyspace_name=f'ks00{keyspace}')
-                table_name.append(f'ks00{keyspace}.table00{table}')
-                # FIXME: improve the structure + data insertion
-                # can use this function to populate tables better?
-                # self.populate_data_parallel()
+        with self.db_cluster.cql_connection_patient(self.db_cluster.nodes[0]) as session:
+            for keyspace in range(num_ks):
+                session.execute(f"CREATE KEYSPACE IF NOT EXISTS ks00{keyspace} "
+                                "WITH replication={'class':'SimpleStrategy', 'replication_factor':1}")
+                for table in range(num_table):
+                    session.execute(f'CREATE COLUMNFAMILY IF NOT EXISTS ks00{keyspace}.table00{table} '
+                                    '(key varchar, c varchar, v varchar, PRIMARY KEY(key, c))')
```

Comment:
Why did you replace create_keyspace and create_table?

Reply:
I was working very very slow, as the commit description suggest.
each one of those calls were option a new connection to the DB, for each table, that's x1000. (and even more since the create_keyspace was checking that keyspace available on one more node, which mean opening/closing 1020 connections)  
like this it works much much faster.